final_prep/README.md

```diff
@@ -32,24 +32,24 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+- <img width="1005" alt="Screen Shot 2021-05-28 at 11 13 15 AM" src="https://user-images.githubusercontent.com/83726180/120021064-d0e45280-bfa7-11eb-8bb4-d1702ff4e2e5.png">
+- <img width="1149" alt="Screen Shot 2021-05-28 at 11 13 33 AM" src="https://user-images.githubusercontent.com/83726180/120020839-82cf4f00-bfa7-11eb-895e-d44a4777d88a.png">
```

Comment:
Nice work here- I like seeing the color coding. I think you are planning on the appropriate amount of work time, though I would expect to work on Saturdays. For me, an approach where I tried to do some work on both Saturday and Sunday worked best so I didn't feel like I was cramming too much into one day.